rust/kcl-lib/src/std/extrude.rs

```diff
@@ -292,86 +292,13 @@ pub(crate) async fn do_post_extrude<'a>(
         vec![]
     };
 
-    // Face filtering attempt in order to resolve https://github.com/KittyCAD/modeling-app/issues/5328
```

Comment:
github wont't let me comment, but a few lines up is
```rust
let solid3d_info = args
        .send_modeling_cmd(
            exec_state.next_uuid(),
            ModelingCmd::from(mcmd::Solid3dGetExtrusionFaceInfo {
                edge_id: any_edge_id,
                object_id: sketch.id,
            }),
        )
        .await?;
```
Should this be removed too @jessfraz

Reply:
if you dont need it we dont need it!

Reply:
hmm actually looks like it's used in `analyze_faces` might just leave it for now.

Reply:
we still need that fwiw